src/BSD-3-Clause.xml

```diff
@@ -28,10 +28,10 @@
             </item>
             <item>
                 <bullet>3.</bullet>
-                <alt match="(The name of.+may not)|(Neither the name of.+nor the names of its contributors may)|( Neither the name of.+nor the names of its authors and contributors may)|(Neither the name of.+nor the names of contributors may)|(Neither the name.+nor the names of contributors may)|(The names of its contributors may not)|(The names of any contributors may not)|(None of the names of.+and any contributors may)" name="organizationClause3">
+                <alt match="(The name of.+may not)|(Neither the name of.+nor the names of its contributors may)|( Neither the name of.+nor the names of its authors and contributors may)|(Neither the name of.+nor the names of contributors may)|(Neither the name.+nor the names of contributors may)|(The names of its contributors may not)|(The names of any contributors may not)|(The names.+must not)|(None of the names of.+and any contributors may)" name="organizationClause3">
                  Neither the name of the copyright holder nor the names of its contributors may</alt>
                  be used to endorse or promote products derived from this
-               <alt match="software|work" name="software">software</alt> without specific prior written permission.
+               <alt match="software|work" name="software">software</alt> without <optional>specific</optional> prior written permission. <optional>To obtain permission, contact &lt;email&gt;.</optional>
```

Comment:
@jlovejoy I wasn't on the 2023-06-22 call mentioned [here](https://github.com/spdx/license-list-XML/issues/1773#issuecomment-1602980253). From the notes it looks to me like the agreement was to include the "must not" vs "may not" in line 31 above, but I don't know whether people are agreed to add this "To obtain permission..." line at the end, since if it's optional it'll show up for BSD-3-Clause generally. Can we just make the line 31 change in this PR?